bepasty_cli/cli.py

```diff
@@ -95,9 +159,23 @@ def convert(self, value, param, ctx):
     '-i',
     '--insecure',
     help='Disable SSL certificate validation',
-    is_flag=True)
-def main(token, filename, fname, url, ftype, list_pastes, insecure, lifetime):
+    is_flag=True,
+    default=configuration['insecure'])
+@click.option(
+    '--write-config',
+    help='Write current command line arguments and defaults to configuration file '
+         '{} and exit.'.format(CONFIG_FILENAME),
+    is_flag=True,
+    default=False)
+def main(token, filename, fname, url, ftype, list_pastes, insecure, lifetime, write_config):
     url = url.rstrip("/")
+    if write_config:
+        lifetime = "".join(lifetime)
+        write_configuration_file(setup_config_parser(locals()))
+        sys.exit(0)
+    if not url:
+        print('Please supply the URL of the bepasty server.')
```

Comment:
A click.Exception here leads to a printed traceback before the error message :/
Above (like 84) it just prints the error message, because it is run from within click.command.